google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingHandler.java

```diff
@@ -310,6 +327,10 @@ private LogEntry entryFor(LogRecord record) {
         .addLabel("levelName", level.getName())
         .addLabel("levelValue", String.valueOf(level.intValue()))
         .setSeverity(severityFor(level));
+    if (gaeInstanceId != null) {
+      builder.addLabel("appengine.googleapis.com/instance_name", gaeInstanceId);
+    }
```

Comment:
I think I want to make the log entry enhancement more "pluggable" and do this work from a strategy object. This interacts with a previous PR of yours - https://github.com/GoogleCloudPlatform/google-cloud-java/pull/1385 - which introduced `enhanceLogEntry`. The problem with that method is that if someone overrides that method, then they can't used AsyncLoggingHandler. I think I might do the conversion to a strategy pattern myself and send a PR for it.